fs/xfstests.py.data/README

```diff
@@ -0,0 +1,41 @@
+This is a simple wrapper for running xfstests inside avocado. The steps to get
+started are really simple:
+
+1) Edit the configuration variables on the yaml file as well environment variable
+1.1) The variables 'TEST_DEV' and 'TEST_DIR' are mandatory and should be set to
+     a block device path and mount point path, respectively, that will be used
+     *exclusively* for xfstests. It must have the filesystem of your choice
+     previously created.
+     ex : export TEST_DEV =/dev/vda1 , export TEST_DIR=/mnt
```

Comment:
Why not to set those using the yaml file? The test can then create the environment variables. No strong opinion here, but it's kinda annoying when the test is not self-sufficient. What do you think?